spec/factories/other_assets_declarations.rb

```diff
@@ -0,0 +1,25 @@
+FactoryBot.define do
+  factory :other_assets_declaration do
+    initialize_with { create(:legal_aid_application).other_assets_declaration }
```

Comment:
Is this necessary?

Reply:
I think so.  You can't create an OtherAssetsDeclaration without a LegalAidApplication existing first.  Unless you know of a better way to do it?

Reply:
Have you tried just:
```ruby
FactoryBot.define do
  factory :other_assets_declaration do
    legal_aid_application
  end
end
```